pkg/shell/machines.js

```diff
@@ -584,6 +584,17 @@ define([
         "#024848"
     ];
 
+    module.colors.parse = function parse_color(input) {
+        var div = document.createElement('div');
+        div.style.color = input;
+        var style = window.getComputedStyle(div, null);
+        for (var i = 0; i < style.length; i++) {
+            if (style[i] == "color")
+                return style.getPropertyValue(style[i]);
+        }
+        return div.style.color;
```

Comment:
For browser compatibility. I ran into problems using style.color directly on the result of getComputedStyle() ... and so I read up about browser compatibility. This seems to be the most compatible way. But happy to change it if that's an incorrect assumption.